srcDoc/release_notes.rst

```diff
@@ -62,6 +62,26 @@ following command-line parameters:
     --cpp-build
     --cs-build
 
+What's Fixed in Master
+~~~~~~~~~~~~~~~~~~~~~~
+
+Review Dynamic Data Send() and Receive() operations (#55)
```

Comment:
Improve *Dynamic Data* Send() and Receive() operations (#55)

Reply:
Done